viite-UI/src/less/core/forms.less

```diff
@@ -373,6 +362,40 @@ form.roadAddressProject p {
   align-items: center;
 }
 
+.node-junctions-table-dimension, .node-points-table-dimension {
+  width:100%;
+}
+
+.node-junctions-table-header-border-bottom {
+  border-bottom: 2px solid @gray-light;
+}
+
+.node-points-table-header, .node-junctions-table-header {
+  width: 5.15%;
+  color: #fff;
+  font-weight: bold;
+  font-size: 12px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-align: left;
+}
+
+.node-junctions-table-border-bottom {
+  border-bottom: 1px dashed @gray-light;
```

Comment:
I think these separating lines would be easier for eyes if they were solid instead of dashed.